SYCL/OnlineCompiler/online_compiler_L0.cpp

```diff
@@ -0,0 +1,53 @@
+// REQUIRES: level_zero, level_zero_headers
+
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple -I %level_zero_include_dir -DRUN_KERNELS -lze_loader %s -o %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple -I %level_zero_include_dir -lze_loader %s -o %th.out
+// RUN: %RUN_ON_HOST %th.out
+
```

Comment:
```suggestion
// UNSUPPORTED: cl_options

```

Reply:
Fixed in a857f2c3e3ab1692e991dba0a5be5bb4ae753181